Source/WebCore/css/CSSProperties.json

```
@@ -6150,7 +6150,8 @@
                 "normal",
                 "break-all",
                 "keep-all",
-                "break-word"
+                "break-word",
+                "auto"
```

Can you add a setting for it?
```suggestion
                {
                    "settings-flag": "cssWordBreakAutoEnabled",
                    "value": "auto"
                }
```